test/drafts/classifieds/ClassifiedsV2.test.ts

```diff
@@ -0,0 +1,114 @@
+import { BigNumber } from 'bignumber.js';
+import { should } from 'chai';
+// tslint:disable-next-line:no-var-requires
+const { advanceTimeAndBlock, takeSnapshot, revertToSnapshot } = require('ganache-time-traveler');
+import { ClassifiedsV2Instance, TestERC20MintableInstance, TestERC721MintableInstance } from '../../../types/truffle-contracts';
+
+const ClassifiedsV2 = artifacts.require(
+    './drafts/classifieds/ClassifiedsV2.sol',
+) as Truffle.Contract<ClassifiedsV2Instance>;
+const TestERC20Mintable = artifacts.require(
+    './test/issuance/TestERC20Mintable.sol',
+) as Truffle.Contract<TestERC20MintableInstance>;
+const TestERC721Mintable = artifacts.require(
+    './test/classifieds/TestERC721Mintable.sol',
+) as Truffle.Contract<TestERC721MintableInstance>;
+
+should();
+
+// tslint:disable-next-line no-var-requires
+const { itShouldThrow } = require('./../../utils');
+
+contract('ClassifiedsV2', (accounts) => {
+    let snapshotId: any;
+
+    const poster = accounts[1];
+    const filler = accounts[2];
+
+    let classifiedsV2: ClassifiedsV2Instance;
+    let erc20token: TestERC20MintableInstance;
+    let erc721token: TestERC721MintableInstance;
+
+    beforeEach(async () => {
+        const snapShot = await takeSnapshot();
+        snapshotId = snapShot.result;
+        erc20token = await TestERC20Mintable.new();
+        erc721token = await TestERC721Mintable.new();
+        classifiedsV2 = await ClassifiedsV2.new(erc20token.address, erc721token.address);
+    });
+
+    afterEach(async  () => {
+        await revertToSnapshot(snapshotId);
+    });
+
+    /**
+     * @test {Classifieds#openTrade}
+     */
+    it('openTrade can succefully open a new trade', async () => {
```

Comment:
This only tests if openTrade emits those events. If the code is wrong the trade might not have been opened. Keep the test, renamed to 'openTrade emits an event to signal trade opening'.

The next test is the one that actually tests `openTrade(...)` and `getTrade(...)`.